src/app/api/attestations/service.ts

```diff
@@ -125,3 +171,25 @@ async function addTeamNameToAttestation(
   }
   return attestations;
 }
+
+function rankContributors(
+  contributors: ContributorDto[],
+): ContributorDataDto[] {
+  return contributors
+    .sort((a, b) => b.contributionScore - a.contributionScore)
+    .map((contributor, index) => {
+      return {
+        githubUsername: contributor.userName,
+        rank: index + 1,
+        score: contributor.contributionScore,
+      };
+    });
+}
+
+function sumContributorScore(contributors: ContributorDto[]) {
+  const sum: number = contributors.reduce(
+    (acc, curr) => acc + curr.contributionScore,
+    0,
+  );
+  return sum;
+}
```

Comment:
I didn't followed through from the already existing functions, but wanted to make a quick note that we need to remember to remove the BOT user type when summing contribution scores otherwise it could be different than what is showing on the UI

Reply:
Yeah, this is already done.  We use the same data as private attestations